### PR TITLE
Set env variables explicitly

### PR DESCRIPTION
**Pull request by zerobfd**
kpt doesn't merge the maps, so we should set all of the env variable for pilot explicitly

### CODE REVIEW

File: asmcli/lib/config.sh [hunk 1]
```diff
@@ -26,6 +26,7 @@ configure_package() {
     kpt cfg set asm gcloud.compute.network "${GCE_NETWORK_NAME}"
   else
     kpt cfg set asm gcloud.core.project "${FLEET_ID}"
+    kpt cfg set asm anthos.servicemesh.controlplane.monitoring.enabled "false"
```
Comment (jtrbs) [inline asmcli/lib/config.sh:29]: This part may not be true. In the case ! is_gcp and "${CA}" = "meshca", kpt cfg set asm anthos.servicemesh.controlplane.monitoring.enabled "true". With Hub registration, for onprem, stackdriver is supported.

Comment (zerobfd) [reply asmcli/lib/config.sh]: Thanks, I didn't know that! Changed to `if ! is_gcp && ! is_cluster_registered`

**Review by ZhengzheYang — APPROVED** [review]

File: asmcli/lib/config.sh [hunk 2]
```diff
@@ -26,7 +25,9 @@ configure_package() {
     kpt cfg set asm gcloud.compute.network "${GCE_NETWORK_NAME}"
   else
     kpt cfg set asm gcloud.core.project "${FLEET_ID}"
-    kpt cfg set asm anthos.servicemesh.controlplane.monitoring.enabled "false"
+    if ! is_cluster_registered; then
```
Comment (jtrbs) [inline asmcli/lib/config.sh:28]: Maybe checking CA is better? cluster registration will be required thus this condition will not be met. However, in the case of Citadel, within ASM source code, stackdriver will not be setup. Thus kpt cfg set asm anthos.servicemesh.controlplane.monitoring.enabled "true" in this case maybe misleading.

Comment (zerobfd) [reply asmcli/lib/config.sh]: Changed, thanks again.

Comment (jtrbs) [reply asmcli/lib/config.sh]: No problem!

**Review by jtrbs — APPROVED** [review]

**Pull request merged**